cassandra_check/conf.yaml.example

```diff
@@ -0,0 +1,16 @@
+init_config:
```

Comment:
Can you fix the format of the config file? Comments should go before actual options and config values can be part of the commented option, for example:
```
# command or path to nodetool (e.g. /usr/bin/nodetool or docker exec container nodetool)
# nodetool: /usr/bin/nodetool

# the list of keyspaces to monitor
# - keyspaces: []
```